cmd/cert-checker/main_test.go

```diff
@@ -478,7 +478,6 @@ func TestIgnoredLint(t *testing.T) {
 	// missing OCSP url in the template.
 	expectedProblems := []string{
 		"zlint error: e_sub_cert_aia_does_not_contain_ocsp_url",
-		"zlint warn: w_serial_number_low_entropy",
```

Comment:
Why is this removed?

Reply:
The lint was removed from the upstream because it necessarily causes false positives. With that lint removed it's output shouldn't be expected anymore.

Reply:
https://github.com/zmap/zlint/pull/292

Reply:
https://github.com/zmap/zlint/pull/292